packages/ui/src/components/search/SearchResult.tsx

```diff
@@ -0,0 +1,169 @@
+import type { APISearchResult } from '@answeroverflow/api';
+import {
+	Message,
+	MessageContents,
+	MessageImages,
+	MessageRenderer,
+	MessageTitle,
+} from '~ui/components/primitives/Message';
+import { Button } from '../primitives/Button';
+
+export interface MessageResultProps {
+	result: APISearchResult[number];
+}
+
+const ServerInviteTitle = ({ name }: { name: string }) => {
+	return (
+		<h3 className="pt-2 text-center font-header text-2xl font-bold text-ao-white">
+			{name}
+		</h3>
+	);
+};
+
+const ServerInviteChannelName = ({ channelName }: { channelName: string }) => {
+	return (
+		<h5 className="text-center text-xl font-light text-ao-white/[.9]">
+			#{channelName}
+		</h5>
+	);
+};
+
+const ServerInviteIcon = () => {
+	return (
+		<div className="h-24 w-24 rounded-[50%] border-2 border-white bg-[#9A9A9A]" />
+	);
+};
+
+const ServerInviteJoinButton = () => {
+	return (
+		<Button type={'solid'} color={'white'} className="my-4">
+			Join Server
+		</Button>
+	);
+};
+
+// TODO: Use data from the API
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const SearchResultMetaData = ({ result }: MessageResultProps) => {
+	return (
+		<div className="mt-2 flex flex-row items-center justify-center">
+			{/* Chat */}
+			<span className="flex items-center justify-center px-1 text-white/[.55]">
+				{0}
+			</span>
+			<svg
```

Comment:
Make this SVG its own component, its hard to read w/ all this in here

Reply:
Done